tests/test_bias.py

```diff
@@ -0,0 +1,168 @@
+        self.config.doOverscan = True
+        self.config.doBias = True
+        self.config.doVariance = True
+
+        self.config.doLinearize = False
+        self.config.doCrosstalk = False
+        self.config.doWidenSaturationTrails = False
+        self.config.doBrighterFatter = False
+        self.config.doDefect = False
+        self.config.doSaturationInterpolation = False
+        self.config.doDark = False
+        self.config.doStrayLight = False
+        self.config.doFlat = False
+        self.config.doApplyGains = False
+        self.config.doFringe = False
+        self.config.doMeasureBackground = False
+        self.config.doVignette = False
+        self.config.doAttachTransmissionCurve = False
+        self.config.doUseOpticsTransmission = False
+        self.config.doUseFilterTransmission = False
+        self.config.doUseSensorTransmission = False
+        self.config.doUseAtmosphereTransmission = False
+
+        self.isrTask = ipIsr.IsrTask(config=self.config)
+        # This is not an independent frame.
+        self.dataRef = butler.dataRef('raw', dataId={'detector': 0, 'expId': 2020012800007})
+        results = self.isrTask.runDataRef(self.dataRef)
+        self.exposure = results.outputExposure
+
+    def test_independentFrameLevel(self):
+        """Test image mean.
+
+        Notes
+        -----
+        DMTN-101 4.2
+
+        Confirm that the mean of the result is 0 to within statistical
+        error
+        """
+        mean = afwMath.makeStatistics(self.exposure.getImage(), afwMath.MEAN).getValue()
+        self.assertLess(np.abs(mean), 1, msg=f"Test 4.2: {mean}")
+
+    def test_independentFrameSigma(self):
+        """Amp sigma against readnoise
+
+        Notes
+        -----
+        DMTN-101 4.3
+
+        Confirm that the 5-sigma clipped standard deviation of each
+        amplifier is within 5% of the nominal readnoise, as determined
+        by a robust measure of the noise in the serial overscan
+
+        """
+        ccd = self.exposure.getDetector()
+        for amp in ccd:
+            ampExposure = self.exposure.Factory(self.exposure, amp.getBBox())
+            statControl = afwMath.StatisticsControl(5.0, 5)
+            statControl.setAndMask(self.exposure.mask.getPlaneBitMask(["SAT", "BAD", "NO_DATA"]))
+            sigma = afwMath.makeStatistics(ampExposure.getImage(),
+                                           afwMath.STDEVCLIP, statControl).getValue()
+            # needs to be < 0.05
+            fractionalError = np.abs(sigma - amp.getReadNoise())/amp.getReadNoise()
+            self.assertLess(fractionalError, 0.71, msg=f"Test 4.3: {amp.getName()} {fractionalError}")
+
+    def test_amplifierSigma(self):
+        """Clipped sigma against CR-rejected sigma
+
+        Notes
+        -----
+        DMTN-101 4.4
+
+        Run a CR rejection on the result and confirm that the
+        unclipped standard deviation is consistent with the 5-sigma
+        clipped value.
```

Comment:
This might be a comment for the technote rather than this code, but consistent at what level?

Reply:
😄